docs/2.9.0/docs/query/pqs-user-guide.rst

```diff
@@ -531,10 +537,21 @@ The ``-pipeline-ledger-start`` argument is an enum with the following possible v
 -  ``Genesis``: Use the first original offset of the ledger. This causes PQS to try to start from offset ``0``. It allows you to load historic creates, archives or (optionally) exercises from a ledger that already has data on it. If you try to restart on an already populated database in this mode, PQS will rewrite data if it needs to.
 -  ``Oldest``: Use the oldest available (unpruned) offset on the ledger or resume where it left off.
 
-PQS is able to start and finish at prescribed ledger offsets, specified by the
-arguments ``--pipeline-ledger-start`` and ``--pipeline-ledger-stop``. The
-``./scribe.jar pipeline --help-verbose`` command provides extensive help
-information.
+--pipeline-ledger-stop
+-----------------------
+
+The ``-pipeline-ledger-stop`` argument is an enum with the following possible values:
+
+-  ``Latest``: Stop reading stream when latest offset is reached.
+-  ``Never``: Continue reading stream indefinitely.
+
+--pipeline-datasource
+---------------------
+
+The ``-pipeline-datasource`` argument is an enum with the following possible values:
+
+-  ``TransactionStream``: Read the ledger's filtered transaction stream and related reassignments for a set of parties. For transactions it lists only creates and archives, but no other events. Omits all events on transient contracts, i.e., contracts that were both created and archived in the same transaction.
```

Comment:
"i.e." needs to be replaced with "that is"

Reply:
reworded